docs/application/tizen-studio/release-notes/3-7-release-notes.md

```diff
@@ -0,0 +1,55 @@
+# Tizen Studio 3.7 Release Notes
+
+- Release Date: Mar 25, 2020
+
+## IDE and Tools
+
+### New Features
+
+- Bundeled JDK with Tizen Studio removing the dependency on system Java
+- Enhaced the security of certificates by adding support for SHA512 encryption
+- Support for Chrome 80 added
```

Comment:
It recommanded to see the style guide: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md
However, the release notes of Tizen Studio has own style. 
So I hope you keep the Tizen Studio style.

The previous release was written as below:
- IDE
  - 5.5 platform images have been updated.
  - **type** property support for Web Runtime (WRT) service applications has been added.

After see the above style, please change the writing style.